lib/housing-wanted/housing-wanted.models.js

```diff
@@ -0,0 +1,45 @@
+const Joi = require('joi');
+
+module.exports = {
+  db: (sequelize, Sequelize) => {
+    const HousingWanted = sequelize.define('HousingWanted', {
+      id: {
+        type: Sequelize.UUID,
+        defaultValue: Sequelize.UUIDV4,
+        primaryKey: true
+      },
+      beds: Sequelize.INTEGER,
+      baths: Sequelize.DECIMAL,
+      contact: Sequelize.STRING,
+      body: Sequelize.TEXT,
+      status: Sequelize.STRING,
+      details: {
+        type: Sequelize.JSONB,
+        defaultValue: {},
+        "cost": {
+          value: Sequelize.DECIMAL,
+        },
+        "security deposit": {
+          value: Sequelize.INTEGER,
+        },
+        "application fee": {
+          value: Sequelize.INTEGER,
+        },
+        "furnished": {
+          value: Sequelize.BOOLEAN,
+        },
```

Comment:
these should not have been added. as far as the database cares, this is just a jsonb field. won't matter what's inside of it. we'll control the structure through api validation and the controllers logic. please remove.